src/warzone/src/warzone/service/RouterService.java

```diff
@@ -149,7 +149,19 @@ public void route(Router p_router) throws IOException{
 					break;
 				case "assigncountries":
 					d_gamePhase.assigncountries();
-					break;						
+					break;
+				case "m":
+					d_gamePhase.setTournamentMapFiles(p_router.getActionParameters().split("\\s+"));
+					break;
+				case "p":
+					d_gamePhase.setTournamentPlayerStrategies(p_router.getActionParameters().split("\\s+"));
+					break;
+				case "g":
+					d_gamePhase.setTournamentNumberOfGames(Integer.parseInt(p_router.getActionParameters()));
+					break;
```

Comment:
the simply solution is to add only one tournament command, 
and the rest of the command will be the parameters send to the related function